404.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="icon" type="image/png" href="images\favicon.png" />
+    <link rel="stylesheet" href="css/style.css" type="text/css" media="screen">
+    <link rel="stylesheet" href="css/semantic.min.css" type="text/css">
+    <title>Error 404</title>
+</head>
+<body>
+    <div class="ui sidebar vertical menu">
+        <a class="item" href="index.html">Home</a>
+        <a class="item" href="whatis.html">Features</a>
+        <a class="item" href="history.html">History</a>
+        <a class="item" href="https://openwisp.io/docs">Docs</a>
+        <a class="item" href="support.html">Support</a>
+        <a class="item" href="thecode.html">Code</a>
+        <a class="item" href="about.html">People</a>
+    </div>
+    <div class="pusher">
+        <div class="ui grid" id="top-bar">
+            <div class="ui computer only column sixteen wide">
+                <div class="ui center aligned container secondary menu">
+                    <h1 class="item logo">
+                        <a href="./">OpenWISP</a>
+                    </h1>
+                    <div class="right menu">
+                        <a class="item" href="index.html">Home</a>
+                        <a class="item" href="whatis.html">Features</a>
+                        <a class="item" href="history.html">History</a>
+                        <a class="item" href="https://openwisp.io/docs">Docs</a>
+                        <a class="item" href="support.html">Support</a>
+                        <a class="item" href="thecode.html">Code</a>
+                        <a class="item" href="about.html">People</a>
+                    </div>
+                </div>
+            </div>
+            <div class="ui mobile tablet only sixteen wide column">
+                <div class="ui menu">
+                    <h1 class="item logo">
+                        <a href="./">OpenWISP</a>
+                    </h1>
+                    <div class="right menu">
+                        <div class="menu-open item">
+                            <i class="sidebar icon"></i>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div align="center">
```

Comment:
you're not following the structure of the other pages, please see how other content pages have "<article>" and the classes they usea and follow the same structure.